src/content/en/fundamentals/instant-and-offline/web-storage/indexeddb-best-practices-app-state.md

```diff
@@ -0,0 +1,160 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Learn best practices for syncing application state between IndexedDB an popular state management libraries
+
+{# wf_updated_on: 2017-06-08 #}
+{# wf_published_on: 2017-06-08 #}
+
+# IndexedDB Best Practices for Persisting Application State {: .page-title }
```

Comment:
Suggested title: "Persisting Application State with IndexedDB" - focus on the what someone is going to accomplish vs the technology they're going to do it with. If you re-title, don't forget to update `_toc.yaml`